src/components/refreshTimeButton/refreshTimeButton.tsx

```diff
@@ -7,7 +7,7 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { Button } from '@patternfly/react-core';
 import { RebootingIcon } from '@patternfly/react-icons';
-import { helpers } from '../../common/helpers';
+import { helpers } from 'src/helpers/helpers';
```

Comment:
lets be consistent here, you used relative paths in some of the other components for `helpers` so this line should be
```
import { helpers } from '../../helpers';
```